frontend/src/AccountDetailsTile.tsx

```diff
@@ -240,38 +327,52 @@ export class AccountDetailsTile extends React.Component<AccountDetailsProps, Acc
+                            </span>}
+                        </span>}</div>
+                    <div>
+                        <div className="btn btn-icon btn-add btn-header" onClick={(_e) => this.addCredential()}>
+                            <div className="align-middle">{ImageProvider.getImage("add-icon")}<span className="btn-header-text">Add Credential</span></div>
+                        </div>
+                        <div className="btn-small inline-block align-middle" onClick={this.close.bind(this)}>✖︎</div>
+                    </div>
                 </div>
                 <div className="box-content">
                     <div className="box-content">
                         <div className="credential">
                             <div className="credential-label">Display Name: </div>
                             <input placeholder="Enter a display name." className="credential-field textarea monospace-font credential-field" onChange={o => this.updateProperty(o.currentTarget.id, o.currentTarget.value)} id="displayName" type="text" value={this.state.account.displayName} />
-                            <div className="btn-icon btn-credentials btn-save" onClick={o => this.saveDisplayName()}>{ImageProvider.getImage("save-icon")}</div>
+                            {displayNameSaveButton}
                         </div>
                         <div className="credential">
                             <div className="credential-label">Platform: </div>
-                            <select value={this.state.account.platformId} placeholder="Enter a platform." id="platformId" className="credential-field-select textarea monospace-font" onChange={o => this.updateProperty(o.currentTarget.id, o.currentTarget.value)}>
+                            <select value={this.state.account.platformId} placeholder="Enter a platform." id="platformId" className="credential-field-select textarea monospace-font" onChange={o => this.prepareCredentials(o.currentTarget.value)}>
                                 <option value="0">&nbsp;</option>
                                 <option value="1">Facebook</option>
                                 <option value="2">Twitter</option>
                             </select>
-                            <div className="btn-icon btn-credentials btn-save" onClick={o => this.saveDisplayName()}>{ImageProvider.getImage("save-icon")}</div>
+                            {platformSaveButton}
                         </div>
+                        {
+                            this.state.account.platformId && this.state.account.platformId === 1 ? this.createInfo("https://confluence.supersocial.cloud/display/SP/Connecting+with+a+Facebook+Page") :
+                                (this.state.account.platformId === 2 &&
+                                    this.createInfo("https://confluence.supersocial.cloud/display/SP/Connecting+with+a+Twitter+Account"))
```

Comment:
Could  this be reduced to a helper like this:
```suggestion
                                    this.createInfo(SnippetManager.confluenceLink("Connecting+with+a+Twitter+Account")))
```

Reply:
Implemented with commit 997968d3c99efee847741328a3b57c511a8027ca